skynet/modules/stt/streaming_whisper/connection_manager.py

```diff
@@ -25,7 +25,10 @@ def __init__(self):
 
     async def connect(self, websocket: WebSocket, meeting_id: str, auth_token: str | None):
         if not bypass_auth:
-            authorized = await authorize(auth_token)
+            jwt_token = utils.get_jwt(websocket.headers, auth_token)
+            authorized = await authorize(
```

Comment:
Why this style with the 3 lines and dangling comma, when it fits in a si gel one?